src/components/TaskStatusIcon/index.tsx

```diff
@@ -0,0 +1,51 @@
+import { TaskStatus } from "types/task";
+import { CannotRun } from "./CannotRun";
+import { Failed } from "./Failed";
+import { KnownFailure } from "./KnownFailure";
+import { Running } from "./Running";
+import { SetupFailure } from "./SetupFailure";
+import { Success } from "./Success";
+import { SystemFailure } from "./SystemFailure";
+import { TimedOut } from "./TimedOut";
+import { WillNotRun } from "./WillNotRun";
+import { WillRun } from "./WillRun";
+
+interface TaskStatusIconProps {
+  status: TaskStatus;
+}
+
+export const TaskStatusIcon: React.FC<TaskStatusIconProps> = ({ status }) => {
```

Comment:
We still need to confirm some of these status/icon mappings. I added it to the design meeting agenda for 8/5